website/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+/node_modules
+
+# Production
+/build
+
+# Generated files
+.docusaurus
+.cache-loader
+
+# Misc
+.DS_Store
+.env.local
```

Comment:
Can we do it like this?
```suggestion
.env
.env*
```

Reply:
This file is auto-generated by toolchain, I think we should keep it as it is for now